### ISSUE TITLE
pasefiguredivs fatal error 

### ISSUE THREAD

**Issue by cderv** [quarto-dev/quarto-cli#6878]
It is probably related : after 1.4.358 (so since 1.4.366 and after), label: tbl-xxxx raises an error when rendered.

a qmd to reproduce the error:

````
---
title: "table"
format: html
---

```{r}
#| label: tbl-unetable

library(gt)
start_date <- "2010-06-07"
end_date <- "2010-06-14"
sp500 |>
  dplyr::filter(date >= start_date & date <= end_date) |>
  dplyr::select(-adj_close) |>
  gt() |>
  tab_header(
    title = "S&P 500",
    subtitle = glue::glue("{start_date} to {end_date}")
  ) |>
  fmt_currency() |>
  fmt_date(columns = date, date_style = "wd_m_day_year") |>
  fmt_number(columns = volume, suffixing = TRUE)
```
````

_Originally posted by @xtimbeau in https://github.com/quarto-dev/quarto-cli/issues/6871#issuecomment-1723222225_

**Comment by cderv** [comment]
```
FATAL (C:/Users/chris/Documents/DEV_R/quarto-cli/src/resources/filters/./quarto-pre/parsefiguredivs.lua:121) An error occurred:
This is an internal error. Please file a bug report at https://github.com/quarto-dev/quarto-cli/
Error running filter C:/Users/chris/Documents/DEV_R/quarto-cli/src/resources/filters/main.lua:
.../DEV_R/quarto-cli/src/resources/filters/./common/log.lua:30: attempt to call a nil value (global 'crash_with_stack_trace')
stack traceback:
        ...EV_R/quarto-cli/src/resources/filters/./common/error.lua:14: in function 'fail'
        ...EV_R/quarto-cli/src/resources/filters/./common/error.lua:19: in function 'internal_error'
        ...i/src/resources/filters/./quarto-pre/parsefiguredivs.lua:121: in function <...i/src/resources/filters/./quarto-pre/parsefiguredivs.lua:70>
        (...tail calls...)
        ...V_R/quarto-cli/src/resources/filters/./common/pandoc.lua:52: in function <...V_R/quarto-cli/src/resources/filters/./common/pandoc.lua:46>
        (...tail calls...)
        [C]: in ?
        [C]: in method 'walk'
        ...R/quarto-cli/src/resources/filters/./ast/customnodes.lua:61: in function <...R/quarto-cli/src/resources/filters/./ast/customnodes.lua:51>
        (...tail calls...)
        .../quarto-cli/src/resources/filters/./ast/runemulation.lua:36: in local 'callback'
        .../quarto-cli/src/resources/filters/./ast/runemulation.lua:49: in upvalue 'run_emulated_filter_chain'
        .../quarto-cli/src/resources/filters/./ast/runemulation.lua:84: in function <.../quarto-cli/src/resources/filters/./ast/runemulation.lua:81>
stack traceback:
        ...R/quarto-cli/src/resources/filters/./ast/customnodes.lua:61: in function <...R/quarto-cli/src/resources/filters/./ast/customnodes.lua:51>
        (...tail calls...)
        .../quarto-cli/src/resources/filters/./ast/runemulation.lua:36: in local 'callback'
        .../quarto-cli/src/resources/filters/./ast/runemulation.lua:49: in upvalue 'run_emulated_filter_chain'
        .../quarto-cli/src/resources/filters/./ast/runemulation.lua:84: in function <.../quarto-cli/src/resources/filters/./ast/runemulation.lua:81>
ERROR: Error
    at renderFiles (file:///C:/Users/chris/Documents/DEV_R/quarto-cli/src/command/render/render-files.ts:334:23)
    at eventLoopTick (ext:core/01_core.js:181:11)
    at async render (file:///C:/Users/chris/Documents/DEV_R/quarto-cli/src/command/render/render-shared.ts:98:18)
    at async Command.fn (file:///C:/Users/chris/Documents/DEV_R/quarto-cli/src/command/render/cmd.ts:212:26)
    at async Command.execute (file:///C:/Users/chris/Documents/DEV_R/quarto-cli/src/vendor/deno.land/x/cliffy@v0.25.4/command/command.ts:1790:7)
    at async quarto (file:///C:/Users/chris/Documents/DEV_R/quarto-cli/src/quarto.ts:135:3)
    at async file:///C:/Users/chris/Documents/DEV_R/quarto-cli/src/quarto.ts:167:5
```

**Comment by cscheid** [comment]
I can repro, am fixing this right now. Thanks for the report!

**Comment by cscheid** [comment]
Fixed by #6887.
